traildb/traildb.py

```diff
@@ -362,17 +414,26 @@ def get_item_value(self, item):
         """Return the string value corresponding to an item."""
         value = lib.tdb_get_item_value(self._db, item, self._uint64_ptr)
         if value is None:
-            raise TrailDBError("Error reading value, error: %s" % lib.tdb_error(self._db))
-        return value[0:self._uint64_ptr.contents.value]
+            raise TrailDBError("Error reading value, error: %s" %
+                               lib.tdb_error(self._db))
+
+        # this is for handle bynary data i.e. test/test.py TEST=test_binarydata
+        # if value can not be converted to unicode we handle it as binary
+        # TODO: find a better way to handle binary data.
```

Comment:
maybe we could add a flag `unicode=True` which tries to decode by default but it can be disabled if you know your tdb contains non-UTF-8 values